packages/tui/internal/theme/catppuccin.go

```diff
@@ -0,0 +1,300 @@
+		Dark:  lipgloss.Color(mauve),
+		Light: lipgloss.Color(lightMauve),
+	}
+	theme.MarkdownLinkColor = compat.AdaptiveColor{
+		Dark:  lipgloss.Color(blue),
+		Light: lipgloss.Color(lightBlue),
+	}
+	theme.MarkdownLinkTextColor = compat.AdaptiveColor{
+		Dark:  lipgloss.Color(sapphire),
+		Light: lipgloss.Color(lightSapphire),
+	}
+	theme.MarkdownCodeColor = compat.AdaptiveColor{
+		Dark:  lipgloss.Color(green),
+		Light: lipgloss.Color(lightGreen),
+	}
+	theme.MarkdownBlockQuoteColor = compat.AdaptiveColor{
+		Dark:  lipgloss.Color(yellow),
+		Light: lipgloss.Color(lightYellow),
+	}
+	theme.MarkdownEmphColor = compat.AdaptiveColor{
+		Dark:  lipgloss.Color(pink),
+		Light: lipgloss.Color(lightPink),
+	}
+	theme.MarkdownStrongColor = compat.AdaptiveColor{
+		Dark:  lipgloss.Color(peach),
+		Light: lipgloss.Color(lightPeach),
+	}
+	theme.MarkdownHorizontalRuleColor = compat.AdaptiveColor{
+		Dark:  lipgloss.Color(overlay1),
+		Light: lipgloss.Color(lightOverlay1),
+	}
+	theme.MarkdownListItemColor = compat.AdaptiveColor{
+		Dark:  lipgloss.Color(blue),
+		Light: lipgloss.Color(lightBlue),
+	}
+	theme.MarkdownListEnumerationColor = compat.AdaptiveColor{
+		Dark:  lipgloss.Color(sapphire),
+		Light: lipgloss.Color(lightSapphire),
+	}
+	theme.MarkdownImageColor = compat.AdaptiveColor{
+		Dark:  lipgloss.Color(lavender),
+		Light: lipgloss.Color(lightLavender),
+	}
+	theme.MarkdownImageTextColor = compat.AdaptiveColor{
+		Dark:  lipgloss.Color(sky),
+		Light: lipgloss.Color(lightSky),
+	}
+	theme.MarkdownCodeBlockColor = compat.AdaptiveColor{
+		Dark:  lipgloss.Color(text),
+		Light: lipgloss.Color(lightText),
+	}
+
+	// Syntax highlighting colors
+	theme.SyntaxCommentColor = compat.AdaptiveColor{
+		Dark:  lipgloss.Color(subtext0),
+		Light: lipgloss.Color(lightSubtext0),
+	}
+	theme.SyntaxKeywordColor = compat.AdaptiveColor{
+		Dark:  lipgloss.Color(mauve),
+		Light: lipgloss.Color(lightMauve),
+	}
+	theme.SyntaxFunctionColor = compat.AdaptiveColor{
+		Dark:  lipgloss.Color(blue),
+		Light: lipgloss.Color(lightBlue),
+	}
+	theme.SyntaxVariableColor = compat.AdaptiveColor{
+		Dark:  lipgloss.Color(flamingo),
+		Light: lipgloss.Color(lightRed),
+	}
+	theme.SyntaxStringColor = compat.AdaptiveColor{
+		Dark:  lipgloss.Color(green),
+		Light: lipgloss.Color(lightGreen),
+	}
+	theme.SyntaxNumberColor = compat.AdaptiveColor{
+		Dark:  lipgloss.Color(peach),
+		Light: lipgloss.Color(lightPeach),
+	}
+	theme.SyntaxTypeColor = compat.AdaptiveColor{
+		Dark:  lipgloss.Color(yellow),
+		Light: lipgloss.Color(lightYellow),
+	}
+	theme.SyntaxOperatorColor = compat.AdaptiveColor{
+		Dark:  lipgloss.Color(sky),
+		Light: lipgloss.Color(lightSky),
+	}
+	theme.SyntaxPunctuationColor = compat.AdaptiveColor{
+		Dark:  lipgloss.Color(overlay2),
+		Light: lipgloss.Color(lightOverlay2),
+	}
+
+	return theme
+}
+
+func init() {
+	// Register the Catppuccin Mocha theme with the theme manager
+	RegisterTheme("catppuccin", NewCatppuccinMochaTheme())
```

Comment:
I think you should call it `catppuccin-mocha` in case the other variants get added

```suggestion
	RegisterTheme("catppuccin-mocha", NewCatppuccinMochaTheme())
```